lib/matplotlib/backends/backend_agg.py

```diff
@@ -29,11 +29,10 @@
 from math import radians, cos, sin
 
 import numpy as np
-from PIL import Image
+from PIL import Image  # noqa: F401 # pylint: disable=W0611
```

Comment:
just remove the import?

Reply:
Yes. I thought removing  it caused some local problems, but I was probably wrong.